dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisPipeline.java

```diff
@@ -220,6 +221,9 @@ private void checkKey(final String key) {
 			 */
 			String hashValue = StringUtils.substringBetween(key, Character.toString(hashtag.charAt(0)),
 					Character.toString(hashtag.charAt(1)));
+			if (Strings.isNullOrEmpty(hashValue)) {
```

Comment:
Shouldn't we rather just check if the pipeline is using hashtags at pipeline creation time and return an error saying that the Dynomite config has hashtags enabled, and therefore so should the pipeline itself?

Reply:
hashtags are set in the config and hashtags are checked during pipeline operations. So, this cannot be checked at pipeline creation time.